controller/api/destination/watcher/endpoints_watcher.go

```diff
@@ -1144,11 +1181,13 @@ func diffAddresses(oldAddresses, newAddresses AddressSet) (add, remove AddressSe
 		}
 	}
 	add = AddressSet{
-		Addresses: addAddresses,
-		Labels:    newAddresses.Labels,
+		Addresses:          addAddresses,
+		Labels:             newAddresses.Labels,
+		LocalTrafficPolicy: newAddresses.LocalTrafficPolicy,
 	}
 	remove = AddressSet{
-		Addresses: removeAddresses,
+		Addresses:          removeAddresses,
+		LocalTrafficPolicy: oldAddresses.LocalTrafficPolicy,
```

Comment:
I don't think this field is necessary for removes

Reply:
Updated